### PR TITLE
Fix indentation

### PR DESCRIPTION
**Pull request by rajbos**
### Why:
Fix indentation in a list:
image

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@rajbos Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ At the end of the month, {% data variables.product.prodname_dotcom %} calculates
 For example, if your organization uses {% data variables.product.prodname_team %} and allows unlimited spending, using 5,000 minutes could have a total storage and minute overage cost of $56 USD, depending on the operating systems used to run jobs.
 
 - 5,000 (3,000 Linux and 2,000 Windows) minutes = $56 USD ($24 USD + $32 USD).
```
Comment (skedwards88) [inline content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md:90]:
```suggestion
- 5,000 (3,000 Linux and 2,000 Windows) minutes = $56 USD ($24 USD + $32 USD):
```

File: content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md [hunk 2]
```diff
@@ -88,8 +88,8 @@ At the end of the month, {% data variables.product.prodname_dotcom %} calculates
...
-  - 3,000 Linux minutes at $0.008 USD per minute = $24 USD.
-  - 2,000 Windows minutes at $0.016 USD per minute = $32 USD.
+- 3,000 Linux minutes at $0.008 USD per minute = $24 USD.
+- 2,000 Windows minutes at $0.016 USD per minute = $32 USD.
```
Comment (skedwards88) [inline content/billing/managing-billing-for-github-actions/about-billing-for-github-actions.md:92]:
```suggestion
  - 3,000 Linux minutes at $0.008 USD per minute = $24 USD.
  - 2,000 Windows minutes at $0.016 USD per minute = $32 USD.
```

**Review by skedwards88 — CHANGES_REQUESTED** [review]
Thanks for opening this PR! This indentation is intentional in order to show the breakdown of the 5000 minutes. Maybe a `:` after the first item can help clarify that? I added suggestions to demonstrate what I mean.

**Comment by rajbos** [comment]
I completely missed that. Still very subtle. Is it an idea to remove the first bullet on the 5000 minute line, end with a colon and then do a normal bulletted list? I think that would make it a lot more clear.

**Pull request closed without merging**